### DISCUSSION TITLE
Welcome to clean-css Discussions!

### DISCUSSION THREAD

**Discussion by jakubpawlowicz** [jakubpawlowicz/clean-css#1134] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of clean-css community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community build together 💪.

To get started, comment below with an introduction of yourself and whatever comes to your mind.

And above all else, please be nice to everyone here.

**Comment by danny-ro** [comment]
Hello, 
i have a problem after i use clean-css the order of the media query is not ok , so i have xs sm lg and after md ,
or sometimes in in the revers not mobile first but desktop first and for bootsrat grid it is the same the orders of the columns is in revers media query desktop first.

Thank you

**Comment by danny-ro** [comment]
I forgot to tell that before using the clean-css the order and the css file is all good the order is mobile first so xs sm md lg.
After the minify it is broken.

TX

**Comment by jakubpawlowicz** [comment]
Hey @danny-ro - please tell me three things: 1) what version of clean-css do you use, 2) how is it executed (I presume by some build step) and 3) can you share a snippet of your CSS which is problematic.

**Comment by danny-ro** [comment]
the clean css ver is 5.3.1

**Comment by jakubpawlowicz** [comment]
@danny-ro and what do you say about no 2 and no 3 above? Please [open an issue](https://github.com/clean-css/clean-css/issues/new) anyway.
